process/sync/baseSync.go

```diff
@@ -840,10 +840,11 @@ func (boot *baseBootstrap) shouldAllowRollback(currHeader data.HeaderHandler, cu
 	finalBlockHash := boot.forkDetector.GetHighestFinalBlockHash()
 	isRollBackBehindFinal := currHeader.GetNonce() <= finalBlockNonce
 	isFinalBlockRollBack := currHeader.GetNonce() == finalBlockNonce
+	wasBlockCommitted := boot.wasBlockCommitted(currHeader)
```

Comment:
misleading name for method, wasBlockCommitted(header) does not verify that the header given as parameter was committed (not doing what it says).

maybe rename to: `canRollbackBlock` ?

Reply:
renamed to canRollbackBlock